config.js

```diff
@@ -0,0 +1 @@
+export const basename = '/isearch/0.14/index.html';
```

Comment:
could you not derive this by doing `require('pacakge.json').version.split('.')[2]` to avoid having to change it on every release...?